JustAnotherVoiceChat.Server.GTMP.Resource/Server/VoiceScript.VoiceEvents.cs

```diff
@@ -89,9 +91,7 @@ private void OnClientRejected(IGtmpVoiceClient client, StatusCode statusCode)
         }
 
         private void OnClientPrepared(IGtmpVoiceClient client)
-        {
-            _voiceServer.NativeWrapper.SetClientVoiceRange(client, 15);
-            
+        {           
```

Comment:
Remove explicit `OnClientPrepared` handler and set the eventhandler to `OnHandshakeShouldResend`